src/radiation/monte_carlo.cpp

```diff
@@ -419,6 +419,9 @@ TaskStatus MonteCarloTransport(MeshBlock *pmb, MeshBlockData<Real> *rc,
               v(prho, k, j, i), v(itemp, k, j, i), v(iye, k, j, i), s, nu);
 
           Real dtau_abs = alphanu * dt; // c = 1 in code units
+          Real Ucon[4];
+          Real vel[3] = {v(ivlo, k, j, i), v(ivlo + 1, k, j, i), v(ivlo + 2, k, j, i)};
+          GetFourVelocity(vel, geom, CellLocation::Cent, k, j, i, Ucon);
```

Comment:
Don't need full four-velocity so you can just compute the 0th component
```suggestion
          Real vel[3] = {v(ivlo, k, j, i), v(ivlo + 1, k, j, i), v(ivlo + 2, k, j, i)};
          Real W = GetLorentzFactor(v, geom, CellLocation::Cent, k, j, i);
          Real alpha = geom.Lapse(CellLocation::Cent, k, j, i);
          Real Ucon0 = robust::ratio(W, std::abs(alpha));
```

Reply:
Fair that's cleaner, will do